backup/moodle2/backup_pdfannotator_stepslib.php

```diff
@@ -54,7 +54,8 @@ protected function define_structure() {
 
         // 2. Define each element separately.
         $pdfannotator = new backup_nested_element('pdfannotator', array('id'), array(
-            'name', 'intro', 'introformat', 'usevotes', 'useprint', 'useprintcomments', 'use_studenttextbox', 'use_studentdrawing', 'useprivatecomments', 'useprotectedcomments', 'timecreated', 'timemodified'));
+            'name', 'intro', 'introformat', 'usevotes', 'useprint', 'useprintcomments', 'use_studenttextbox', 'use_studentdrawing',
+            'useprotectedcomments', 'useprivatecomments', 'forcesubscribe', 'timecreated', 'timemodified'));
```

Comment:
Hi @lucaboesch ,
is there a reason why you changed the order of 'useprivatecomments' and 'useprotectedcomments'?

Reply:
Oh, sorry, Friederike, that's a copy-past error.
It was of course meant to stay the same.

Reply:
I amended the patch.